test/org/freedesktop/gstreamer/PropertyTypeTest.java

```diff
@@ -150,30 +150,33 @@ public void setEnum() {
         audiotestsrc.setAsString("wave", "Silence");
         assertEquals(4, audiotestsrc.get("wave"));
         assertEquals("Silence", audiotestsrc.getAsString("wave"));
-        
+
         audiotestsrc.setAsString("wave", "square");
         assertEquals(1, audiotestsrc.get("wave"));
         assertEquals("Square", audiotestsrc.getAsString("wave"));
-        
+
         audiotestsrc.setAsString("wave", "red-noise");
         assertEquals(10, audiotestsrc.get("wave"));
         String redNoise = audiotestsrc.getAsString("wave");
         assertEquals("Red (brownian) noise", redNoise);
         audiotestsrc.setAsString("wave", redNoise);
         assertEquals(10, audiotestsrc.get("wave"));
-        
+
         // invalid value
         audiotestsrc.set("wave", -256);
         assertEquals(0, audiotestsrc.get("wave"));
     }
-    
+
     @Test(expected = IllegalArgumentException.class)
     public void setEnumInvalidString() {
         audiotestsrc.setAsString("wave", "FOO");
     }
-    
+
     @Test
     public void setValueArrayFromString() {
+        if (!Gst.testVersion(1, 14)) {
```

Comment:
Event with GStreamer 1.16 installed, this will never be true, becuase the actual version is limited to the requested version on Gst.init which in this Test is not specified and therefore defaults to `Version.BASELINE` which is only 1.8.0; so this Test will never be true, even when a more modern GStreamer is installed.

Reply:
Damn!  Yes, thanks, forgot to change the init line - in general tests should be using the pattern from https://github.com/gstreamer-java/gst1-java-core/blob/master/test/org/freedesktop/gstreamer/PromiseTest.java#L37

Reply:
Actually the String-Array-Initialization in the linked Init-Call is unnecessary; `Gst.init(Gst.getVersion(), "PromiseTest");` resolves to the same Constructor.

These are the Things that IntelliJ suggests. You might want to use a better IDE ;) :P (just kidding.)

Reply:
:laughing: Yes, I didn't write that one - sure the author was probably using Eclipse! :wink:

Key thing is use of `Gst.getVersion()` in future.

Reply:
I opened #191 fixing the things discussed here.